File: 005-FizzBuzz/005-matt-fizzbuzz.c
#include<stdio.h>
#include<stdlib.h>
#include<stdint.h>

int main(void){
	uint8_t i;
	for(i=1; i<=100; i++){
        if(i%15==0){
            puts("FizzBuzz");
        }else if(i%5==0){
            puts("Buzz");
        }else if(i%3==0){
            puts("Fizz");
        }else{
            printf("%d\n", i);
        }
	}
	return EXIT_SUCCESS;
}

File: 005-FizzBuzz/005-matt-fizzbuzz.rs
fn main() {
	for i in 1..101 {
		fizzbuzz(i);
	}
}

fn fizzbuzz(x: i32) {
	match (x % 3, x % 5) {
		(0, 0) => println!("fizzbuzz"),
		(_, 0) => println!("buzz"),
		(0, _) => println!("fizz"),
		_  => println!("{}", x),
	}
}
